src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -790,6 +791,75 @@ public UnspentCoinModel[] ListUnspent(int minConfirmations = 1, int maxConfirmat
             return unspentCoins.ToArray();
         }
 
+        [ActionName("listtransactions")]
+        [ActionDescription("Returns an array of transactions belonging to this wallet.")]
+        public TransactionInfoModel[] ListTransactions(string label = "*", int count = 10, int skip = 0)
```

Comment:
Should we have the `include_watchonly` parameter as well? As we do make a distinction between transactions in 'normal' accounts vs special accounts. And it makes sense to make the parameter list match as closely as possible in general.